### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by jasalisbury**
**Description of change**
 * Report table context menu for approved reports changed from "edit" to "view"
 * Error message state lifted up to the activity report
 * Not editable alert for submitted reports duplicated below "reset to draft" button
 * Accordion items auto expanded for non-draft/needs action reports
 * Resources used is now a `textarea`
 * Title and other verbiage tweaked for next steps
 * Report status tag capitalized and green if report is approved

**How to test**
Pull down or visit [sandbox](https://tta-smarthub-sandbox.app.cloud.gov/)

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/392

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

**Review by dcmcand — COMMENTED** [review]
Is this correct? Both say "What have you agreed to do next?"

Screenshot from 2021-03-16 15-46-48

**Comment by jasalisbury** [comment]
> Is this correct? Both say "What have you agreed to do next?"
> 
> Screenshot from 2021-03-16 15-46-48

Yeah it isn't correct, I read the issue wrong, working on an update.

**Review by dcmcand — APPROVED** [review]
:tada:

**Comment by rahearn** [comment]
> * Report table context menu for approved reports changed from "edit" to "view"

Is this duplicative of #250?

**Comment by jasalisbury** [comment]
> > * Report table context menu for approved reports changed from "edit" to "view"
> 
> Is this duplicative of #250?

Yeah but that PR fixes legacy links and has some style changes. We can either merge that one through to main or into this branch

**Comment by rahearn** [comment]
> We can either merge that one through to main or into this branch

Looks like they both had the same frontend failure too, why don't you merge #250 into this one and I'll review them together.

File: frontend/src/pages/Landing/index.css [hunk 1]
```diff
@@ -147,6 +147,11 @@ h1.landing {
   background-color: #148439;
 }
 
+.landing .smart-hub--context-menu-button {
+  max-width: 20px;
```
Comment (rahearn) [inline frontend/src/pages/Landing/index.css:151]: I think this is causing the button to look weird:

Screen Shot 2021-03-17 at 9 32 45 AM

**Review by rahearn — CHANGES_REQUESTED** [review]
Looks good, save the one styling fix.

While you're at it, I have one other tweak that I think should be easy but wasn't in the original ticket. Can you hide the "Edit" links on the review page for reports that aren't editable? I think it should be the same logic as deciding whether to expand the accordion or not.

**Comment by rahearn** [comment]
> Can you hide the "Edit" links on the review page for reports that aren't editable?

If this is more than a one line fix then please open a new JIRA ticket for it and let me know.

**Comment by jasalisbury** [comment]
> > Can you hide the "Edit" links on the review page for reports that aren't editable?
> 
> If this is more than a one line fix then please open a new JIRA ticket for it and let me know.

It's more than a one line fix, I'll create a ticket in JIRA

**Review by rahearn — APPROVED** [review]
🎉

**Pull request merged**